packages/lesswrong/components/common/HomeLatestPosts.jsx

```diff
@@ -9,6 +9,12 @@ import { withStyles } from '@material-ui/core/styles';
 import {AnalyticsContext, captureEvent} from '../../lib/analyticsEvents';
 
 const styles = theme => ({
+  personalBlogpostsCheckbox: {
+    // Hackily counteract margin from SectionFooterCheckbox
+    // We probably shouldn't be using SectionFOOTERCheckbox in the SectionTitle,
+    // but will probably refactor soon so won't bother fixing.
+    marginBottom: -16,
```

Comment:
...does this work on desktop? It seems like it'd look off-center there

Reply:
Good spotting, that should only be on mobile because the SectionFooterCheckbox margin is only on mobile. However it doesn't affect the centering, as (my explanation) the margin intruding into the text doesn't cause it to rise, the text just overflows beyond the margin.